README.md

```diff
@@ -1,9 +1,45 @@
 # PHP Releases Action
-Docker container action to consume the PHP Releases API for use within GitHub Actions.
+Generates an array of supported PHP releases for use within a GitHub Actions matrix.
 
-## Requirements
+## Usage
+Within your workflow `jobs` block, add the following job _before_ your matrix job:
+```yaml
+  output_releases:
+    name: Generate PHP Releases Array
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Current Releases
+        uses: tighten/phpreleases-action@v1
+        id: releases
+```
+If you need to add specific versions to the matrix statically (to ensure they're included regardless of current support), add them as a comma separated list with the `releases` key:
+```yaml
+  output_releases:
+    name: Generate PHP Releases Array
+    runs-on: ubuntu-latest
+    steps:
+      - name: Fetch Current Releases
+        uses: tighten/phpreleases-action@v1
+        id: releases
+        with:
+          releases: '7.4, 7.3'
+```
 
-## Installation
+A full sample is available in this repo's [.github/workflows directory](https://github.com/tighten/phpreleases-action/blob/main/.github/workflows/main.yml).
+
+Then, refer to the `output_releases` job's output in the `php` line of your matrix strategy, like below:
+```yaml
+  strategy:
+    matrix:
+      php: ${{ fromJSON(needs.output_releases.outputs.range) }}
+```
+
+You can implement additional matrix values (os or laravel versions, for example), as well as include/exclude combinations [as you normally would with a matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs).
+
```

Comment:
I copied over our RAM entry example to include as a visual representation of what you were saying.

```suggestion
## Example
```yaml
name: Run Tests

on:
  push:
    branches: [ main ]
  pull_request:

jobs:
  output_releases:
    name: Generate PHP Releases Array
    runs-on: ubuntu-latest
    steps:
      - name: Fetch Current Releases
        uses: tighten/phpreleases-action@v1
        id: releases
        with:
          releases: '7.4'
  tests:
    needs: output_releases
    strategy:
      matrix:
        os: [Ubuntu, Windows, macOS]
        php: ${{ fromJSON(needs.output_releases.outputs.range) }}

        include:
        - os: Ubuntu
          os-version: ubuntu-latest

        - os: Windows
          os-version: windows-latest

        - os: macOS
          os-version: macos-latest

    name: ${{ matrix.os }} - PHP ${{ matrix.php }}

    runs-on: ${{ matrix.os-version }}

    steps:
    - name: Checkout code
      uses: actions/checkout@v1

    - name: Setup PHP
      uses: shivammathur/setup-php@v2
      with:
          php-version: ${{ matrix.php }}
          extensions: posix, dom, curl, libxml, mbstring, zip, pcntl, bcmath, soap, intl, gd, exif, iconv, imagick
```

```